server/sonar-server/src/test/java/org/sonar/server/organization/ws/SearchActionTest.java

```diff
@@ -191,47 +192,65 @@ public void request_returns_only_specified_keys_ordered_by_createdAt_when_filter
     insertOrganization(ORGANIZATION_DTO.setUuid("uuid5").setKey("key-5"));
     insertOrganization(ORGANIZATION_DTO.setUuid("uuid4").setKey("key-4"));
 
-    assertThat(executeRequest(1, 10, "key-3", "key-1", "key-5"))
+    assertThat(executeRequestAndReturnList(1, 10, "key-3", "key-1", "key-5"))
       .extracting(Organization::getKey)
       .containsExactly("key-5", "key-1", "key-3");
     // ensure order of arguments doesn't change order of result
-    assertThat(executeRequest(1, 10, "key-1", "key-3", "key-5"))
+    assertThat(executeRequestAndReturnList(1, 10, "key-1", "key-3", "key-5"))
       .extracting(Organization::getKey)
       .containsExactly("key-5", "key-1", "key-3");
+  }
 
-    // verify paging
-    assertThat(executeRequest(1, 1, "key-1", "key-3", "key-5"))
-      .extracting(Organization::getKey)
-      .containsExactly("key-5");
-    assertThat(executeRequest(1, 2, "key-1", "key-3", "key-5"))
-      .extracting(Organization::getKey)
-      .containsExactly("key-5", "key-1");
-    assertThat(executeRequest(2, 2, "key-1", "key-3", "key-5"))
-      .extracting(Organization::getKey)
-      .containsExactly("key-3");
+  @Test
+  public void result_is_paginated() {
+    when(system2.now()).thenReturn(SOME_DATE, SOME_DATE + 1_000, SOME_DATE + 2_000, SOME_DATE + 3_000, SOME_DATE + 5_000);
+    insertOrganization(ORGANIZATION_DTO.setUuid("uuid3").setKey("key-3"));
```

Comment:
remark: I like to use `IntStream.rangeClosed(1, 5).forEach(i -> )` when I repeat the same operation several times